test/fw/ptl/lib/pbs_testlib.py

```diff
@@ -6469,22 +6460,9 @@ def manager(self, cmd, obj_type, attrib=None, id=None, extend=None,
                        i.e. silent mode
         :type logerr: bool
         :raises: PbsManagerError
-        :returns: On success:
-                - if expect is False, return code of qmgr/pbs_manager
-                - if expect is True, 0 for success
-        :raises: On Error/Failure:
                 - PbsManagerError if qmgr/pbs_manager() failed
```

Comment:
line 6462 already mentions "raises: PbsManagerError", so I think we can remove this line as well